### PR TITLE
fix(typo): clarify "vasty" in help "lost 7" is not a typo

### PR DESCRIPTION
**Pull request by MCOfficer**
**Bugfix:** This PR addresses issue #6054

## Fix Details
`vasty` -> `vast`

## Testing Done
/

## Save File
/

lol indeed

### CODE REVIEW

**Comment by Amazinite** [comment]
https://github.com/endless-sky/endless-sky/pull/5103#discussion_r429571612

**Comment by tehhowch** [comment]
Let's add a comment to mark that this is intentional

File: data/help.txt [hunk 1]
```diff
@@ -62,6 +62,7 @@ help "lost 5"
 help "lost 6"
 	`You slip deeper and deeper into the zenlike calm of outer space. The Tao which can be spoken is not the true Tao. The game that must be played is not the true game. You let go of all desire: your desire to press <Show main menu> and check the game's preferences to find out how to control your ship. The desire to view the other places you can travel to by pressing <View star map>. You release even the desire to press <Initiate hyperspace jump> and initiate a random hyperjump in whatever direction you are facing in. Enlightenment hovers on the edge of your consciousness, so close you can almost grasp it.`
 
+# Note to future proofreaders: "vasty" is a reference to Firefly, see http://signal.serenityfirefly.com/mmx/segment/view.php/2253
```
Comment (tehhowch) [inline data/help.txt:65]:
```suggestion
# Note to future proofreaders: "vasty nothingness" is a reference to Firefly, see http://signal.serenityfirefly.com/mmx/segment/view.php/2253
```

**Review by tehhowch — COMMENTED** [review]
Thanks

**Pull request merged**